runtime_builders/template_builder.py

```diff
@@ -48,10 +48,16 @@ def _resolve_and_publish(config_file, bucket):
     try:
         gcs_paths = []
         with open(config_file, 'r') as f:
-            project_cfg = json.load(f)
+            if 'json' in config_file:
```

Comment:
I'd rather stick with just one for simplicity unless we've actually gotten requests for both.